index.js

```diff
@@ -18,51 +19,24 @@ module.exports = {
       ],
     });
 
-    var chartistTreeSCSS = new Funnel(chartistPath, {
-      srcDir: 'scss',
-
-      include: [
-        '*.scss',
-      ],
-    });
-
-    return new MergeTrees([
+    return mergeTrees([
       vendorTree,
       chartistTree,
-      chartistTreeSCSS,
     ]);
   },
 
+  treeForStyles() {
+    var chartistPath = path.dirname(require.resolve('chartist'));
```

Comment:
I was maintaining consistency with the code currently in `master`, but I can swap over to let/const if you'd prefer.